Kernel/Heap/SlabAllocator.cpp

```diff
@@ -130,6 +131,7 @@ void slab_alloc_init()
     s_slab_allocator_16.init(128 * KB);
     s_slab_allocator_32.init(128 * KB);
     s_slab_allocator_64.init(512 * KB);
+    s_slab_allocator_64.init(512 * KB);
```

Comment:
Not initializing the right bucket here ;)

Reply:
Whoops, not sure what bad things this caused...